stable/rocketchat/templates/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: nginx
```

Comment:
It should be commented out by default. Similar to https://github.com/kubernetes/charts/blob/e80d1cfb8857cc71096b3b8a4638a789b378db88/stable/grafana/values.yaml#L23

This will allow the Chart to work by default. People that use the nginx-based ingress controller can either opt-in by setting this to "nginx" or they configure their cluster/ingress-controller to pick up Ingresses without annotations (like GKE does).